<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fUsuarios</class>
 <widget class="QDialog" name="fUsuarios">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="forms.qrc">
    <normaloff>:/img/favicon.ico</normaloff>:/img/favicon.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Datos Personales</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <layout class="QVBoxLayout" name="verticalLayoutTodosDatosPersonales">
         <item>
          <widget class="QFrame" name="frameNombre">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 127);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="font">
                   <font>
                    <pointsize>10</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>DNI/NIE/NIF:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="txtDni">
                  <property name="font">
                   <font>
                    <pointsize>10</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="inputMask">
                   <string/>
                  </property>
                  <property name="maxLength">
                   <number>11</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="font">
                   <font>
                    <pointsize>10</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Primer Apellido:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="txtApellido1">
                  <property name="font">
                   <font>
                    <pointsize>10</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="font">
                   <font>
                    <pointsize>10</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Segundo Apellido:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="txtApellido2">
                  <property name="font">
                   <font>
                    <pointsize>10</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="font">
                   <font>
                    <pointsize>10</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="text">
                   <string>Nombre:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="txtNombre">
                  <property name="minimumSize">
                   <size>
                    <width>250</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>10</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frameContacto">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(170, 255, 255);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_14">
                <item>
                 <widget class="QLabel" name="label_14">
                  <property name="text">
                   <string>Teléfono:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="txtTelefono">
                  <property name="maxLength">
                   <number>9</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_15">
                <item>
                 <widget class="QLabel" name="label_15">
                  <property name="text">
                   <string>Móvil:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="txtMovil">
                  <property name="maxLength">
                   <number>9</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_16">
                <item>
                 <widget class="QLabel" name="label_16">
                  <property name="text">
                   <string>E-mail:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="txtEmail"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_17">
                <item>
                 <widget class="QLabel" name="label_17">
                  <property name="text">
                   <string>Web:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="txtWeb">
                  <property name="minimumSize">
                   <size>
                    <width>280</width>
                    <height>0</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frameNacionalidad">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(85, 255, 127);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Fecha de nacimiento:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDateEdit" name="txtFechaNacimiento"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Lugar de nacimiento:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="txtLugarNacimiento"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>Nacionalidad:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="txtNacionalidad"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_8">
                <item>
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>Sexo:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="txtSexo">
                  <item>
                   <property name="text">
                    <string>Hombre</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Mujer</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_9">
                <item>
                 <widget class="QLabel" name="label_9">
                  <property name="text">
                   <string>Fecha de alta:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDateEdit" name="txtFechaAlta"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frameDireccion">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 170, 127);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_28">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_10">
                <item>
                 <widget class="QLabel" name="label_10">
                  <property name="text">
                   <string>Código postal:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="txtCodigoPostal"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_11">
                <item>
                 <widget class="QLabel" name="label_11">
                  <property name="text">
                   <string>localidad:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="txtLocalidad"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_12">
                <item>
                 <widget class="QLabel" name="label_12">
                  <property name="text">
                   <string>Municipio:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="txtMunicipio"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_13">
                <item>
                 <widget class="QLabel" name="label_13">
                  <property name="text">
                   <string>Provincia:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="txtProvincia"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_18">
                <item>
                 <widget class="QLabel" name="label_18">
                  <property name="text">
                   <string>Tipo de vía:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="txtTipoVia"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_19">
                <item>
                 <widget class="QLabel" name="label_19">
                  <property name="text">
                   <string>Nombre de vía:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="txtNombreVia">
                  <property name="minimumSize">
                   <size>
                    <width>190</width>
                    <height>0</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_20">
                <item>
                 <widget class="QLabel" name="label_20">
                  <property name="text">
                   <string>Número:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="txtNumero"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_21">
                <item>
                 <widget class="QLabel" name="label_21">
                  <property name="text">
                   <string>Planta:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="txtPlanta"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_22">
                <item>
                 <widget class="QLabel" name="label_22">
                  <property name="text">
                   <string>Puerta:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="txtPuerta"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_23">
                <item>
                 <widget class="QLabel" name="label_23">
                  <property name="text">
                   <string>Bloque:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="txtBloque"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_24">
                <item>
                 <widget class="QLabel" name="label_24">
                  <property name="text">
                   <string>Portal:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="txtPortal"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_25">
                <item>
                 <widget class="QLabel" name="label_25">
                  <property name="text">
                   <string>Escalera:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="txtEscalera"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_26">
                <item>
                 <widget class="QLabel" name="label_26">
                  <property name="text">
                   <string>Km:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="txtKm"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 170, 255);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_27">
              <item>
               <widget class="QLabel" name="label_27">
                <property name="text">
                 <string>Observaciones:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPlainTextEdit" name="txtObservaciones"/>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabDocumentacion">
      <attribute name="title">
       <string>Documentación</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="forms.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>fUsuarios</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>fUsuarios</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
